src/main/java/fi/vm/yti/datamodel/api/v2/repository/BaseRepository.java

```diff
@@ -0,0 +1,82 @@
+package fi.vm.yti.datamodel.api.v2.repository;
+
+import org.apache.jena.arq.querybuilder.AskBuilder;
+import org.apache.jena.graph.NodeFactory;
+import org.apache.jena.query.Query;
+import org.apache.jena.query.QuerySolution;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdfconnection.RDFConnection;
+import org.apache.jena.update.UpdateRequest;
+
+import java.util.function.Consumer;
+
+public class BaseRepository {
```

Comment:
Could this be an abstract class so it does not get instatiated and has to be extended by a class like ImportsRepository

Reply:
Yes, fixed